spec/02-integration/03-admin_api/01-kong_routes_spec.lua

```diff
@@ -1,113 +1,146 @@
+      local json = cjson.decode(body)
+      assert.is_string(json.configuration.pg_password)
+      assert.not_equal("hide_me", json.configuration.pg_password)
+    end)
+    it("returns PRNG seeds", function()
+      local res = assert(client:send {
+        method = "GET",
+        path = "/",
+      })
+      local body = assert.response(res).has.status(200)
+      local json = cjson.decode(body)
+      assert.is_table(json.prng_seeds)
+      for k, v in pairs(json.prng_seeds) do
+        assert.matches("pid: %d+", k)
+        assert.matches("%d+", k)
+      end
+    end)
+  end)
+
+  dao_helpers.for_each_dao(function(kong_conf)
+    describe("/status with DB: " .. kong_conf.database, function()
+      local client
+
+      setup(function()
+        assert(helpers.start_kong {
+          database = kong_conf.database,
         })
-        assert.res_status(200, res)
-        assert.equal(string.format("%s/%s", meta._NAME, meta._VERSION), res.headers.server)
-        assert.is_nil(res.headers.via) -- Via is only set for proxied requests
+        client = helpers.admin_client(10000)
       end)
-      it("returns 405 on invalid method", function()
-        local methods = {"POST", "PUT", "DELETE", "PATCH", "GEEEET"}
-        for i = 1, #methods do
-          local res = assert(client:send {
-            method = methods[i],
-            path = "/",
-            body = {}, -- tmp: body to allow POST/PUT to work
-            headers = {["Content-Type"] = "application/json"}
-          })
-          local body = assert.response(res).has.status(405)
-          local json = cjson.decode(body)
-          assert.same({ message = "Method not allowed" }, json)
-        end
+
+      teardown(function()
+        if client then client:close() end
+        helpers.stop_kong()
       end)
-      it("exposes the node's configuration", function()
+
+      it("returns status info", function()
         local res = assert(client:send {
           method = "GET",
-          path = "/"
+          path = "/status"
         })
         local body = assert.res_status(200, res)
         local json = cjson.decode(body)
-        assert.is_table(json.configuration)
+        assert.is_table(json.database)
+        assert.is_table(json.server)
+
+        assert.is_boolean(json.database.reachable)
+
+        assert.is_number(json.server.connections_accepted)
+        assert.is_number(json.server.connections_active)
+        assert.is_number(json.server.connections_handled)
+        assert.is_number(json.server.connections_reading)
+        assert.is_number(json.server.connections_writing)
+        assert.is_number(json.server.connections_waiting)
+        assert.is_number(json.server.total_requests)
       end)
-      it("obfuscates sensitive settings from the configuration", function()
+
+      it("database.reachable is `true` when DB connection is healthy", function()
+        -- In this test, we know our DB is reachable because it must be
```

Comment:
not a blocker for this PR, but could we simply mock out the dao's `reachable` function to test this?